.github/workflows/build.yml

```
@@ -73,3 +73,16 @@ jobs:
             USERNAME=paf
             USER_UID=1000
             USER_GID=1000
+
+      - name: Save PR ID
+        env:
+          PR_ID: ${{ github.event.number }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:6: Double quote to prevent globbing and word splitting

(shellcheck)